api/controller.py

```diff
@@ -820,6 +826,63 @@ def detail(self, data_source, identifier_type, identifier):
             feed = unicode(feed)
             return feed_response(feed, None)
 
+class AnnotationController(CirculationManagerController):
+
+    def container(self):
+        headers = dict()
+        headers['Allow'] = 'GET,HEAD,OPTIONS,POST'
+        headers['Accept-Post'] = AnnotationWriter.CONTENT_TYPE
+
+        if flask.request.method=='HEAD':
+            return Response(status=200, headers=headers)
+
+        patron = flask.request.patron
+
+        if flask.request.method == 'GET':
+            headers['Link'] = ['<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"',
+                               '<http://www.w3.org/TR/annotation-protocol/>; rel="http://www.w3.org/ns/ldp#constrainedBy"']
+            headers['Content-Type'] = AnnotationWriter.CONTENT_TYPE
```

Comment:
Since you have the timestamps you could do `Last-Modified`. Also see https://github.com/NYPL-Simplified/server_core/pull/388/files#r79421726 for my ETag implementation proposal. Both of these could go in a separate branch.